scripts/check-release.sh

```diff
@@ -0,0 +1,11 @@
+
+# Get the main field from package.json
+FILE=$(cat ./package.json | grep main | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
```

Comment:
This can be slightly simplified:
```suggestion
FILE=$(grep '"main"' package.json | sed 's/.*:\s*"\(.*\)".*/\1/')
```

There's an alternative which will also validate if the json is valid, but only works with python installed (in my case, the command is `python3`): 
```console
cat package.json | python -c 'import json,sys;obj=json.load(sys.stdin);print(obj["main"])';
```

Reply:
Thanks! I reverted back because this command: `echo $(grep '"main"' package.json | sed 's/.*:\s*"\(.*\)".*/\1/')` yields this output for me: `"main": "./build/index.js",`

And I just need the path for the rest of the command to work

Reply:
Strange... I knew sed had different behaviors in different systems. It must be some kind of nuance I'd need to check.

In any case:
```suggestion
FILE=$(grep '"main":' ./package.json | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
```
should work fine